api/rest/client.go

```diff
@@ -0,0 +1,159 @@
+package rest
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Client is an interface that can talk with dcos-diagnostics REST API and manipulate remote bundles
+type Client interface {
+	// Create ask node url to start bundle creation process with given id
+	Create(ctx context.Context, node string, id string) (*Bundle, error)
```

Comment:
```suggestion
	Create(ctx context.Context, node string, ID string) (*Bundle, error)
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms